### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by weiiwang01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,30 @@
 # WordPress Operator
 
-A Juju charm for a Kubernetes deployment of WordPress, configurable to use a
-MySQL backend.
+A Juju charm deploying and managing WordPress on Kubernetes.
+WordPress is the world's most popular website builder, and it's free and open-source.
```
Comment (yanksyoon) [inline README.md:4]: Indico seems to have a link directed to Indico, would having a link to wordpress also help?
i.e.
```suggestion
WordPress is the world's most popular website builder, and it's free and open-source. Find out more about WordPress' features [here](https://wordpress.com/features/).
```

Comment (weiiwang01) [reply README.md:4]: I think they remove that in newer versions of the Indico charm, but adding a link to WordPress official website is a good idea. Added, thanks!

**Review by yanksyoon — DISMISSED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -1,48 +1,30 @@
...
+and gives them the freedom to deploy on the Kubernetes platform of their choice.
 
-## Usage
+For DevOps or SRE teams this charm will make operating WordPress simple and straightforward through Juju's clean interface.
+It will allow easy deployment into multiple environments for testing of changes, 
+and supports scaling out for enterprise deployments.
 
-For details on using Kubernetes with Juju [see here](https://juju.is/docs/kubernetes), and for
-details on using Juju with MicroK8s for easy local testing [see here](https://juju.is/docs/microk8s-cloud).
+Project and community
```
Comment (jdkandersson) [inline README.md:20]: Should this be a heading?

File: README.md [hunk 3]
```diff
@@ -1,48 +1,30 @@
...
 
-To deploy the charm and relate it to the [MariaDB K8s charm](https://charmhub.io/mariadb) within a Juju
-Kubernetes model:
+The Indico Operator is a member of the Ubuntu family.
```
Comment (jdkandersson) [inline README.md:22]: Indico?

File: README.md [hunk 4]
```diff
@@ -1,48 +1,30 @@
...
-    mariadb/0*          active       idle   10.1.234.43  3306/TCP  ready
-    wordpress-k8s/0*    active       idle   10.1.234.13  80/TCP    Pod configured
-
-In this case our `UNIT_IP` is 10.1.234.13. If we visit `http://${UNIT_IP}/`
-you'll see the WordPress site itself, or you can log in to the admin site
-at `http://{$UNIT_IP}/wp-admin` using a username of `admin` and the password
-from the `get-initial-password` action above.
-
-For further details, [see here](https://charmhub.io/wordpress-k8s/docs).
+Thinking about using the Indico Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
```
Comment (jdkandersson) [inline README.md:30]: Indico?

File: README.md (hunk 2 above)
Comment (weiiwang01) [reply README.md]: Fixed, thanks!

File: README.md (hunk 3 above)
Comment (weiiwang01) [reply README.md]: Fixed, thanks!

File: README.md (hunk 4 above)
Comment (weiiwang01) [reply README.md]: Fixed, thanks!

**Review by jdkandersson — APPROVED** [review]

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Pull request merged**